ui/src/app/subnets/views/SubnetsList/_index.scss

```diff
@@ -1,56 +1,58 @@
-.subnets-table {
-  tr:not(:first-child) {
-    border-top: 0 !important;
-  }
-  tbody tr:not(:first-child) td {
-    border-top: 1px solid $color-mid-light;
-  }
+@mixin SubnetsList {
```

Comment:
This was a drive-by fix to include this class in a mixin. Below this line is all indentation changes.